languages/ruby/exercises/concept/hashes/log_line.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
```

Comment:
I think this should be the `.meta/example.rb` file just FYI

Reply:
And perhaps (just a comment) if there are no literal strings to protect this should at least not be `true`.  (I usually don't like it to exist at all, it often lies, and is otherwise scaffolding, and scaffolding is ugly when not being used.)